src/dev/impl/DevToys/Strings/en-US/GZipEncoderDecoder.resw

```diff
@@ -0,0 +1,165 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AccessibleName" xml:space="preserve">
+    <value>GZip Encoder and Decoder tool</value>
+  </data>
+  <data name="Ascii" xml:space="preserve">
+    <value>ASCII</value>
+  </data>
+  <data name="ConfigurationTitle" xml:space="preserve">
+    <value>Configuration</value>
+  </data>
+  <data name="ConversionDecompress" xml:space="preserve">
+    <value>Decompress</value>
+  </data>
+  <data name="ConversionDescription" xml:space="preserve">
+    <value>Select compress or decompress</value>
+  </data>
+  <data name="ConversionCompress" xml:space="preserve">
+    <value>Compress</value>
+  </data>
+  <data name="ConversionTitle" xml:space="preserve">
+    <value>Compress/Decompress</value>
+  </data>
+  <data name="MenuDisplayName" xml:space="preserve">
+    <value>GZip</value>
+  </data>
+  <data name="EncodingDescription" xml:space="preserve">
+    <value>Compress or decompress strings with GZip</value>
+  </data>
+  <data name="EncodingTitle" xml:space="preserve">
+    <value>Encoding</value>
+  </data>
+  <data name="InputTitle" xml:space="preserve">
+    <value>Input</value>
+  </data>
+  <data name="OutputTitle" xml:space="preserve">
+    <value>Output</value>
+  </data>
+  <data name="Utf8" xml:space="preserve">
+    <value>UTF-8</value>
+  </data>
+  <data name="Description" xml:space="preserve">
+    <value>Compress or decompress strings</value>
+  </data>
+  <data name="SearchDisplayName" xml:space="preserve">
+    <value>GZip Compress/Decompress</value>
```

Comment:
Please add a space around `/` to get `Compress / Decompress`. It slightly improves the readability and make it consistent with other tools.

Reply:
Fixed